### DISCUSSION TITLE
NEW TO CODING!?

### DISCUSSION THREAD

**Discussion by ADAM323323** [buildwiththeta/buildwiththeta#116] [Q&A]
Good Morning, 

New to coding here, have a good idea on designing an app so trying to learn how i can develop it myself. However, I'm struggling a lot, I'm not sure how I can add widgets using Theta and can't find any videos or advice how how to use Theta.

Would be grateful for any advice!
Thanks 
Adam

**Comment by andreabuttarelli** [comment]
Hi, I'm Andrea!

Have you had a look at the demo and our documentation yet? 

- Video demo: https://www.youtube.com/watch?v=oFed0NIqBZI
- Documentation: https://docs.buildwiththeta.com

Let me know!
